pkg/translate31to22/translate31to22.go

```diff
@@ -0,0 +1,428 @@
+package translate31to22
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/coreos/ign-converter/pkg/util"
```

Comment:
Appears in an inconsistent place between 30to22 and 31to22, and so makes diff noisier.

Reply:
fixed